api/src/main/java/org/infinispan/api/InfinispanManager.java

```diff
@@ -0,0 +1,29 @@
+package org.infinispan.api;
+
+import org.infinispan.api.configuration.InfinispanClientConfig;
+import org.infinispan.api.configuration.InfinispanConfig;
+
+/**
+ * Entry point for Infinispan API - client
+ *
+ * @author Katia Aresti, karesti@redhat.com
+ * @since 10.0
+ */
+@Experimental
+public class InfinispanManager {
```

Comment:
As the methods in this class are static and essentially just loading implementations, could we not expose them directly in `Infinispan.java`. This way we have one less class for the user to worry about and they can do the following:

```java
Infinispan client = new Infinispan.client(clientConfig);
Infinispan embedded = new Infinispan.embedded(embeddedConfig);
```

Reply:
Totally